scripts/update/index.ts

```diff
@@ -0,0 +1,18 @@
+import * as path from 'path';
+import fs from 'fs-extra';
+import { exit } from 'process';
+import { addVersions } from './tasks/add-versions.js';
+import { readPackages } from './tasks/read-packages.js';
+
+if (!process.argv[2]) {
+  exit(-1);
```

Comment:
It'd be good to write a message to stderr here saying that this needs an argument (and what that argument is)